package.json

```diff
@@ -198,7 +198,12 @@
     "**/@types/react-dom": "^17.0.0",
     "**/cross-fetch": "^3.0.5",
     "**/dot-prop": "^5.2.0",
-    "**/react-is": "^17.0.1"
+    "**/react-is": "^17.0.1",
+    "scheduler": "0.21.0-alpha-73ffce1b6-20210624",
+    "react-dom": "18.0.0-alpha-73ffce1b6-20210624",
+    "react-test-renderer": "18.0.0-alpha-73ffce1b6-20210624",
+    "react-is": "18.0.0-alpha-73ffce1b6-20210624",
+    "react": "18.0.0-alpha-73ffce1b6-20210624"
```

Comment:
Is this change intended? Does it mean we run the whole project with React 18 with this in?